.github/workflows/test.yml

```diff
@@ -104,11 +153,57 @@ jobs:
         with:
           path: ./setup-cli
 
+      - name: Assert databricks CLI is not already installed
+        run: ./setup-cli/assert/not-installed.sh
+        shell: bash
+
       - run: curl -fsSL https://raw.githubusercontent.com/databricks/setup-cli/${{ github.sha }}/install.sh | sh
         shell: bash
 
       - run: databricks version
         shell: bash
 
-      - run: ./setup-cli/test.sh $(cat ./setup-cli/VERSION)
+      - name: Assert the version of the CLI installed
+        run: ./setup-cli/assert/version.sh $(cat ./setup-cli/VERSION)
+        shell: bash
+
+      - name: Assert installation path is /usr/local/bin for non-windows platforms
+        run: ./setup-cli/assert/path.sh /usr/local/bin/databricks
+        if: matrix.os != 'windows-latest'
+        shell: bash
+
+      - name: Assert installation path is /c/Windows for windows platforms
+        if: matrix.os == 'windows-latest'
+        run: ./setup-cli/assert/path.sh /c/Windows/databricks
+
+  curl-dbr:
+    # All DBR images are built on top of Ubuntu
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+        with:
+          path: ./setup-cli
+
+      # Append ~/bin to the PATH. This helps with the assertions in the next steps.
```

Comment:
How does it help?

E.g. "When run on DBR, the installation scripts installs to ~/bin. Add it to $PATH to make assertions work."

Reply:
Addressed in https://github.com/databricks/setup-cli/pull/91